src/components/tl/mlx5/mcast/tl_mlx5_mcast.h

```diff
@@ -33,6 +33,14 @@
 #define DROP_THRESHOLD    1000
 #define MAX_COMM_POW2     32
 
+/* Allgather RDMA-based reliability designs */
+#define ONE_SIDED_RELIABILITY_MAX_TEAM_SIZE 1024
```

Comment:
where is it checked that the team doesn't exceed this size?

Reply:
during mlx5/team create (this is going to be in future PRs)